### DISCUSSION TITLE
One-stop shop Seamless Tutorial

### DISCUSSION THREAD

**Discussion by kauterry** [facebookresearch/seamless_communication#282] [Show and tell]
We have just released our notebook which serves as an exhaustive tutorial to learn how to use the entire suite of Seamless models. This was presented as part of the NeurIPS 2023 - Seamless EXPO workshop. This serves to answer most of the questions the user might have while using the seamless_communication repository.

Please feel free to play with it, you can open it in Google Colab as well.

You can find it here: [Seamless_Tutorial.ipynb](https://github.com/facebookresearch/seamless_communication/blob/main/Seamless_Tutorial.ipynb)

**Comment by Alokkumar8** [comment]
can you help me deploy SeamlessStreaming model to Replicate?
